src/compiler/scala/tools/nsc/typechecker/Implicits.scala

```diff
@@ -360,8 +360,8 @@ trait Implicits {
     val undetParams = if (isView) Nil else context.outer.undetparams
     val wildPt = approximate(pt)
 
-    private val runDefintions = currentRun.runDefinitions
-    import runDefintions._
+    private val currentRunRunDefinitions = currentRun.runDefinitions
+    import currentRunRunDefinitions._
```

Comment:
Ow.  That is not a very friendly name.  If you're going to use so many letters, maybe at least explain why this whole thing is necessary `val stableRunDefsForImport = currentRun.runDefinitions`